.github/workflows/intel-mkl.yml

```diff
@@ -35,7 +35,7 @@ jobs:
   linux-container:
     runs-on: ubuntu-22.04
     container:
-      image: ghcr.io/rust-math/rust-mkl:1.63.0-2020.1
+      image: ghcr.io/rust-math/rust-mkl:1.67.0-2020.1
```

Comment:
Thank you for your work. However, I don't think it will work as there's no tag named 1.67-2020.1 for rust-mkl https://github.com/rust-math/rust-mkl-container/pkgs/container/rust-mkl/versions?filters%5Bversion_type%5D=tagged . I think we should rewrite CI and install intel-mkl by apt command

Besides, I’m not certain if the MSRV policy has been implemented. If it’s permitted to bump MSRV, I could also help finish it. 

cc @bluss

Reply:
Does that mean, we have to update the repository of: https://github.com/rust-math/rust-mkl-container/blob/main/.github/workflows/pack.yml

To provide a docker image with the desired rust version? So we potentially wait for the discussion on MSRV to settle for a specified rust version and than go here for an update in the corresponding rust-mkl repository? Or can we do this already in advance independently to MSRV?